src/core/client/admin/components/UserHistoryDrawer/UserStatusDetailsContainer.tsx

```diff
@@ -22,13 +23,23 @@ interface Props {
 
 const UserStatusDetailsContainer: FunctionComponent<Props> = ({ user }) => {
   const activeBan = useMemo(() => {
-    return user.status.ban.history.find((item) => item.active);
+    if (user.status.ban.active) {
+      return last(user.status.ban.history);
+    }
+    return null;
```

Comment:
this was actually a bug previously, since `user.status.ban.history` entries are not mutated when a ban is lifted, multiple items may be marked `active` even if a ban has been removed.